doc/api/errors.md

```diff
@@ -1579,6 +1584,11 @@ recommended to use 2048 bits or larger for stronger security.
 A TLS/SSL handshake timed out. In this case, the server must also abort the
 connection.
 
+<a id="ERR_TLS_RENEGOTIATE"></a>
+### ERR_TLS_RENEGOTIATE
+
+An attempt was made to renegotiate TLS session failed.
```

Comment:
Is this wording a bit off? Maybe `An attempt made to renegotiate...` or even `An attempt to renegotiate...`?

Reply:
I was not sure, thats why I asked everyone here  [#21440](https://github.com/nodejs/node/issues/21440) about the message

Reply:
`An attempt to renegotiate the TLS session failed`.